openapi.yaml

```diff
@@ -280,6 +280,7 @@ components:
         - $ref: https://raw.githubusercontent.com/asyncapi/spec-json-schemas/master/schemas/2.2.0.json
         - $ref: https://raw.githubusercontent.com/asyncapi/spec-json-schemas/master/schemas/2.3.0.json
         - $ref: https://raw.githubusercontent.com/asyncapi/spec-json-schemas/master/schemas/2.4.0.json
+        - $ref: https://raw.githubusercontent.com/asyncapi/spec-json-schemas/master/schemas/2.5.0.json
```

Comment:
We could just use https://www.asyncapi.com/schema-store/all.schema-store.json and remove the others.

Reply:
Ok, but not in this PR, ok? :)